varinfo/umm_var.py

```diff
@@ -31,9 +31,11 @@
 
 """
 
+from __future__ import annotations
```

Comment:
Sorry this is an annoying thing to be picky about but can we save formatting changes for formatting PRs? It makes it harder to follow. I know I am also guilty of doing this too 😆

Reply:
Yup - agreed. Initially I didn't think this PR would be half as big as it ended up being, so it didn't seem too big a deal to include. But I'll be limiting type hint changes to things that are directly relevant to the other PRs for these schema changes, and saving less relevant ones for separate work. (I do want to get to a point where `mypy` is happy and we can add it to pre-commit without it yelling at us)